README.md

```diff
@@ -23,17 +23,17 @@
 
 ## :framed_picture: Screenshots
 
-### :sun_with_face: Light theme
+### :sun_with_face: Light
 
 | <img src="metadata/en-US/images/phoneScreenshots/home-light.png" width="500" align="center"/> | <img src="metadata/en-US/images/phoneScreenshots/app-light.png" width="500" align="center"/> |
 |:---------------------------------------------------------------------------------------------:|:--------------------------------------------------------------------------------------------:|
 
-### :last_quarter_moon_with_face: Dark theme
+### :last_quarter_moon_with_face: Dark
 
 | <img src="metadata/en-US/images/phoneScreenshots/home-dark.png" width="500" align="center"/> | <img src="metadata/en-US/images/phoneScreenshots/app-dark.png" width="500" align="center"/> |
 |:--------------------------------------------------------------------------------------------:|:-------------------------------------------------------------------------------------------:|
 
-### :waxing_crescent_moon: Amoled theme
+### :waxing_crescent_moon: Black
```

Comment:
Black theme*

Reply:
I agree, this applies to the three themes

Reply:
I think it goes without saying, but OK.